ci/terraform/ecs.tf

```diff
@@ -140,6 +140,10 @@ locals {
         name  = "SERVICE_DOMAIN"
         value = local.service_domain
       },
+      {
+        name  = "SUPPORT_WELSH_LANGAGE_IN_SUPPORT_FORMS"
```

Comment:
Typo in '...LANGAGE...'

Reply:
Thanks for spotting this @CraigRyder . Just pushing a fix now 👍🏽